src/context-menu/CanvasContextMenu.tsx

```diff
@@ -93,7 +134,10 @@ export function getMenuOptionsVisibility(models) {
     let isSingleParameterNodeSelected = parameterNodes.length === 1;
     let isSingleComponentNodeSelected = componentNodes.length === 1;
     let showReloadNode = isNodeSelected && componentNodes.length > 0;
-    let showopenXircuitsWorkflow = isSingleComponentNodeSelected && models.some(model => isXircuitsWorkflow(model))
+    let showopenXircuitsWorkflow = isSingleComponentNodeSelected && models.some(model => isXircuitsWorkflow(model));
+    let showAttachNode = isNodeSelected && parameterNodes.length > 0;
```

Comment:
Do I understand this correctly that this will show up on non literal nodes too?

Reply:
Ah good one. Currently it shows when at least one parameter node is selected, but it should be when at least one literal node is selected as arguments can't be attached. 
One thing I missed was the context being able to banish Literals to the shadow realm if they're not connected. That logic has been fixed.